pkgs/development/libraries/gsignond/default.nix

```diff
@@ -0,0 +1,37 @@
+{ stdenv, fetchFromGitLab, pkgconfig, meson, ninja, glib, glib-networking, sqlite, gobjectIntrospection, vala_0_34 }:
+
+stdenv.mkDerivation rec {
+  pname = "gsignond";
+  version = "1.0.7";
+
+  name = "${pname}-${version}";
+
+  src = fetchFromGitLab {
+    owner = "accounts-sso";
+    repo = pname;
+    rev = version;
+    sha256 = "03a9d92hldav18bhx63kgi88li8fnm56337bjsr7391awq1k9mv7";
+  };
+
+  nativeBuildInputs = [ pkgconfig meson ninja gobjectIntrospection vala_0_34 ];
+
+  buildInputs = [ glib glib-networking sqlite ];
+
+  mesonFlags = [
+    # Building documentation does not work in sandboxed environments
+    "-D documentation=false"
+  ];
+
+  postPatch = ''
+    substituteInPlace meson.build \
+    --replace "sysconf_dir = '/etc'" "sysconf_dir = 'etc/'"
+  '';
+
+  meta = with stdenv.lib; {
+    description = "a D-Bus service which performs user authentication on behalf of it's clients";
+    homepage = https://gitlab.com/accounts-sso/gsignond;
+    license = licenses.lgpl2;
```

Comment:
This should be lgpl21Plus: https://gitlab.com/accounts-sso/gsignond/blob/44e1e3dca19ece7f35ed6ee8c9cfbd7767003dbb/src/daemon/main.c#L13